src/assets/js/scope.js

```diff
@@ -10,12 +10,20 @@ class Scope {
   }
 
   update(user) {
-    const {
-      staffInformation: { staff: { value: isStaff = false } = {} } = {},
-      accessInformation: {
-        mozilliansorg: { values: { nda: isNdaed = false } = {} } = {},
-      } = {},
-    } = user;
+    let isStaff = false;
+    let isNdaed = false;
+    try {
+      const {
+        staffInformation: { staff: { value: isStaffBool = false } = {} } = {},
+        accessInformation: {
+          mozilliansorg: { values: { nda: isNdaedBool = false } = {} } = {},
+        } = {},
+      } = user;
+      isStaff = isStaffBool;
+      isNdaed = isNdaedBool;
+    } catch (e) {
+      console.error('Error: ', e);
+    }
```

Comment:
Yeah, that was broken. However, `{ nda: null }` actually means that someone is in the nda group.

```js
    const {
      staffInformation: { staff: { value: isStaff = false } = {} } = {},
      accessInformation: {
        mozilliansorg: { values: { nda: isNdaed = false } = {} } = {},
      } = {},
    } = user || {};
    this.isStaff = isStaff;
    this.isNdaed = isNdaed === null;
``` 

would solve for that and prevent exceptions since every case has a default.